src/features/OntologyImportBreeder.feature

```diff
@@ -173,3 +173,21 @@ Feature: Ontology Import (10 Scenarios)
 		And user uploads "test_traits_extraCols.xlsx" file
 		And user selects 'Import' button
 		Then user can see 'Confirm New Ontology Term' header
+
+	@BI-1267
```

Comment:
Is there a reason why this is in debug with steps commented out? Is there another card that will address this or is it still in progress?

Reply:
I think it was my mistake.
Looking back at the original against the current implementation, I have to check with the team about the expected behavior.
I will remove this pull request in the meantime.